iron-overlay-behavior.html

```diff
@@ -549,23 +570,113 @@
     },
 
     /**
-     * Executes a callback on the next animation frame, overriding any previous
-     * callback awaiting for the next animation frame. e.g.
-     * `__onNextAnimationFrame(callback1) && __onNextAnimationFrame(callback2)`;
-     * `callback1` will never be invoked.
-     * @param {!Function} callback Its `this` parameter is the overlay itself.
+     * Delays the execution of a callback to the next animation frame.
+     * To cancel a previous job, pass callback = undefined
```

Comment:
Using an undefined second argument the cancel a previous job is subtle and impossible to intuit without reading this comment. Consider factoring raf cancellation into an explicit method.

Reply:
You're right. Actually, this "feature" is not used anywhere, so I'll just expect it to be always defined 👌